### ISSUE TITLE
About training gpu type and training time?

### ISSUE THREAD

**Issue by DRosemei** [chenhsuanlin/bundle-adjusting-NeRF#9]
Thanks for your great work! I want to know about training gpu type and training time for BARF and NeRF.

**Comment by chenhsuanlin** [comment]
Hi @DRosemei, with the default settings, training BARF takes around 9-12 hours on a single Tesla V100 GPU, while for NeRF you can get a 30% speedup (as the gradients don't backprop to the camera poses). Note that the NeRF baseline here doesn't train with coarse/fine networks as with in the original NeRF paper.
Hope this helps!

**Comment by DRosemei** [comment]
> Hi @DRosemei, with the default settings, training BARF takes around 9-12 hours on a single Tesla V100 GPU, while for NeRF you can get a 30% speedup (as the gradients don't backprop to the camera poses). Note that the NeRF baseline here doesn't train with coarse/fine networks as with in the original NeRF paper. Hope this helps!

Thanks for your help!
